### PR TITLE
VB-4331 - Public booker can provide email address for main contact

### PR DESCRIPTION
**Pull request by adaviesMOJ**
## What does this pull request do?

Introduce the ability for a public booker to provide an email address for the visit main contact.

Add new email column to the application and visit contact tables (nullable).

Update all integration tests.

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/visit/UpdateVisitTest.kt [hunk 1]
```diff
@@ -78,7 +78,7 @@ class UpdateVisitTest : IntegrationTestBase() {
     createApplicationAndSave(sessionTemplate = sessionTemplateDefault, completed = false, visitRestriction = visitRestriction)
 
     var expiredReservedApplication = applicationEntityHelper.create(slotDate = existingVisit.sessionSlot.slotDate, sessionTemplate = sessionTemplateDefault, completed = false, visitRestriction = visitRestriction)
-    applicationEntityHelper.createContact(application = expiredReservedApplication, name = "Jane Doe", phone = "01234 098765")
+    applicationEntityHelper.createContact(application = expiredReservedApplication, name = "Jane Doe", phone = "01234 098765", email = "example@email.com")
```
Comment (dhirajshettymoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/visit/UpdateVisitTest.kt:81]: should we replace all instances of example@email.com to email@example.com as email.com is an actual domain?

**Review by dhirajshettymoj — COMMENTED** [review]
Just 1 minor comment about the email used in tests. Otherwise looks good

File: src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/visit/UpdateVisitTest.kt (hunk 1 above)
Comment (adaviesMOJ) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/visit/UpdateVisitTest.kt:81]: yeah I can do this

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Review by dhirajshettymoj — DISMISSED** [review]
LGTM

**Pull request merged**